### PR TITLE
Update pillars-turbomodule.md

### PR DESCRIPTION
**Pull request by ismaelsousa**
Add more info to the process

### CODE REVIEW

File: docs/the-new-architecture/pillars-turbomodule.md [hunk 1]
```diff
@@ -223,7 +260,7 @@ android
         ├── AndroidManifest.xml
         └── java
             └── com
-                └── rtncalculator
+                └── RTNCalculator
```
Comment (cortinico) [inline docs/the-new-architecture/pillars-turbomodule.md:263]: Commenting here but valid for the rest of PR:
I'm against those changes specifically as:
- packages in java/kotlin are all lowercase
- Class prefixes in Java/Kotlin like `RTN` are uncommon. It's a pattern in iOS but not on Android. I'm not sure we want Android to follow the iOS convention. We want instead to follow the platform specific conventions.

**Review by cortinico — CHANGES_REQUESTED** [review]

File: docs/the-new-architecture/pillars-turbomodule.md (hunk 1 above)
Comment (ismaelsousa) [reply docs/the-new-architecture/pillars-turbomodule.md:263]: @cortinico I understand, the problem itself was maintaining the standard of declarations such as class names. RTNCalculatorPackage in one location and CalculatorPackage in another.

Comment (cortinico) [reply docs/the-new-architecture/pillars-turbomodule.md:263]: I understand, but the React Native framework is following the same convention. If you use any scaffodling tool, you would follow the same convention.
The rest of the website, also follows this convention. I'd refrain from breaking it here

Comment (ismaelsousa) [reply docs/the-new-architecture/pillars-turbomodule.md]: Done @cortinico, thank you!

File: docs/the-new-architecture/pillars-turbomodule.md [hunk 2]
```diff
@@ -570,13 +570,13 @@ android
         └── java
             └── com
                 └── RTNCalculator
```
Comment (cortinico) [inline docs/the-new-architecture/pillars-turbomodule.md:572]:
```suggestion
                └── rntcalculator
```

File: docs/the-new-architecture/pillars-turbomodule.md [hunk 3]
```diff
@@ -442,6 +479,21 @@ RCT_REMAP_METHOD(add, addA:(NSInteger)a
 @end
 ```
 
+The final result should look like this:
```
Comment (cortinico) [inline docs/the-new-architecture/pillars-turbomodule.md:482]: I would have only one of this tree view at the end probably?

Comment (ismaelsousa) [reply docs/the-new-architecture/pillars-turbomodule.md:482]: make sense, I'll do it soon

Comment (ismaelsousa) [reply docs/the-new-architecture/pillars-turbomodule.md]: much better now @cortinico 🎉

**Review by cortinico — APPROVED** [review]

**Pull request merged**